.github/workflows/build.yml

```diff
@@ -104,8 +104,33 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
     env:
-      # Set up wheels matrix.  This is CPython 3.6--3.9 for all OS targets.
-      CIBW_BUILD: "cp3[6-9]-*"
+      # Set up wheels matrix.  This is CPython 3.6--3.10 for all OS targets.
+      CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-* cp310-*"
+      # We build with oldest-supported-numpy, see:
+      # https://github.com/scipy/oldest-supported-numpy/blob/master/setup.cfg
+      CIBW_BEFORE_BUILD: "cp pypi-pyproject.toml pyproject.toml"
+      # Explicitly specify the build frontend. We should change this to
+      # "build" when we release QuTiP 4.7.
+      CIBW_BUILD_FRONTEND: "pip"
+      # cibuildwheel 2.3.0 changed the default manylinux image to 2014, but
+      # for now we continue to build against manylinux 2010. When we switch
+      # to 2014, we should also switch to the manylinux_2_17_{arch} tags.
+      CIBW_MANYLINUX_X86_64_IMAGE: "manylinux2010"
+      CIBW_MANYLINUX_I686_IMAGE: "manylinux2010"
+      CIBW_MANYLINUX_PYPY_X86_64_IMAGE: "manylinux2010"
+      CIBW_MANYLINUX_AARCH64_IMAGE: "manylinux2010"
+      CIBW_MANYLINUX_PPC64LE_IMAGE: "manylinux2010"
+      CIBW_MANYLINUX_S390X_IMAGE: "manylinux2010"
+      CIBW_MANYLINUX_PYPY_AARCH64_IMAGE: "manylinux2010"
```

Comment:
These have never supported manylinux2010, the first version to have special archs was 2014. Also, I don't think you should be using 2010 for PyPy unless you really need to, as PyPy wants to drop 2010 soon.